README.md

```diff
@@ -35,35 +35,34 @@ You tell diffjam how to count occurrences of violations of the policies
 by providing strings to search for like `TODO`, and then diffjam counts instances of TODO in the codebase.
 It can also be used in a githook to prevent new cases of a bad pattern.
 
-Adherence to the policies isn't all-or-nothing.  Diffjam can be used in builds to track the number of
+Adherence to the policies isn't all-or-nothing. Diffjam can be used in builds to track the number of
```

Comment:
🤣 I also prefer the double space. :old-man-yells-at-cloud: but MLA was updated. but also Markdown/html completely ignores duplicate spaces, so my editor's markdown prettier/formatting-on-save caught this and just "fixed" it all.

Most of the changes in this file are that auto-fix process :/